libs/langchain-google-cloud-sql-pg/src/vectorStore.ts

```diff
@@ -178,15 +180,86 @@ class PostgresVectorStore extends VectorStore {
     )
   }
 
+  async addVectors(vectors: number[][], documents: DocumentInterface[], options?: { metadatas?: Array<Record<string, any>> | null, ids?: string[] }): Promise<string[] | void> {
+    if (options && (!options.ids || options.ids.length === 0)) {
+      options.ids = Array.from(documents).map(() => uuidv4());
+    }
+
+    if (options && !options.metadatas) {
+      options.metadatas = Array.from(documents).map(() => new Object());
+    }
+
+    const tuples = customZip(options?.ids, documents, vectors, options?.metadatas);
+
+    // Insert embeddings
+    for (const [id, content, embedding, metadata] of tuples) {
+      const metadataColNames = this.metadataColumns.length > 0 ? this.metadataColumns.join(",") : "";
+
+      let stmt = `INSERT INTO "${this.schemaName}"."${this.tableName}"(${this.idColumn}, ${this.contentColumn}, ${this.embeddingColumn}, ${metadataColNames}`
+      let values = {
+        id: id,
+        content: content.pageContent,
```

Comment:
To clarify the above comment, we can keep content here but refer to the document variable for the pageContent
```suggestion
        content: document.pageContent,
```

Reply:
Done